libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.html

```diff
@@ -0,0 +1 @@
+<div #chartContainer [style.height]="height + 'px'"></div>
```

Comment:
See https://github.com/kirbydesign/designsystem/blob/master/libs/designsystem/src/lib/components/chart/chart.component.ts on how to get rid of redundant div

Reply:
Done.